File: tests/fixtures/func_sample.js
// module level - excluded with functions-only
const MODULE_STR = "module_string";
const MODULE_NUM = 200;

// Regular function
function myFunc() {
    const x = "func_string";
    const n = 99;
    return x;
}

// Arrow function
const arrowFn = (param) => {
    const msg = "arrow_string";
    return msg;
};

// Nested control flow inside function (literals still included)
function withControl() {
    if (true) {
        const inside = "control_inside";
    }
}

File: tests/fixtures/func_sample.py
"""Module docstring with 'excluded_module' value."""

MODULE_STR = "module_string"
MODULE_NUM = 100


def my_func() -> str:
    """Function docstring."""
    x = "func_string"
    n = 42
    return x


class MyClass:
    class_var = "class_var"

    def method(self) -> None:
        """Method docstring."""
        msg = "method_string"
        count = 7


async def async_func() -> None:
    """Async function docstring."""
    value = "async_string"
    num = 55
